Binary Build Fails if Project Path Contains Spaces

I've been attempting to install the `deasync` package but been running into problems doing so.

The problem seems to happen when `g++` executes during the build process and tries to load the `node-addon-api` package from the `node_modules` directory.

Given that my project path is:
`/mnt/c/Other Projects/babel-async-traversal`

I get the following error during the installation process:
```
make: Entering directory `/mnt/c/Other Projects/babel-async-traversal/node_modules/deasync/build'
  CXX(target) Release/obj.target/deasync/src/deasync.o
g++: error: Projects/babel-async-traversal/node_modules/node-addon-api: No such file or directory
make: *** [Release/obj.target/deasync/src/deasync.o] Error 1
make: Leaving directory `/mnt/c/Other Projects/babel-async-traversal/node_modules/deasync/build'
gyp ERR! build error
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/home/USER/.nvm/versions/node/v11.13.0/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:262:23)
gyp ERR! stack     at ChildProcess.emit (events.js:193:13)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:255:12)
```

Note that `g++` is searching for a directory called `Projects/babel-async-traversal/node_modules/node-addon-api` and finding that it doesn't exist.  It sure doesn't.  It seems that at some point, the path gets split at the space character of the `Other Projects` directory.

Searching for a possible cause, [this issue](https://github.com/formulahendry/vscode-code-runner/issues/248) suggests that the path should be wrapped in double-quotes before passing it to `g++` to ensure that it takes the string as a single path, and not two paths delimited by a space.

A small change in [this file](https://github.com/abbr/deasync/blob/master/binding.gyp), where the location of the `node-addon-api` package is determined by executing Node, is likely all that is needed to resolve this issue.

--- a/binding.gyp
+++ b/binding.gyp
@@ -22,7 +22,7 @@
         "src/deasync.cc"
       ],
       "include_dirs": [
-        "<!@(node -p \"require('node-addon-api').include\")"
+        "<!@(node -p \"require('node-addon-api').include.replace(/\\\s/g, \\\"\\\\\\\\\\\\\\\ \\\")\")"
       ],
       "conditions": [
         [

